docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: '3.8'
+
+networks:
+    web:
+      external: true
+      driver:  bridge
+
+services:
+  cla-end-to-end:
+    build: .
+    volumes:
+      - .:/behave
+    networks:
+      - web
+    ports:
+      - "8002:80"
+    links:
+      - clabackend
+      - clafrontend
+      - seleniumchrome
+
+  seleniumchrome:
+    image: selenium/standalone-chrome:latest
+    hostname: chrome
+    networks:
+      - web
+    ports:
+      - "5900:5900"
+      - "4444:4444"
+    privileged: true
+    shm_size: 2g
```

Comment:
What does this do?

Reply:
priviledged has been removed, the SHM is the shared memory which has been specified as needed for the selenium chrome